themes/default/content/blog/building-developer-portals/index.md

```diff
@@ -0,0 +1,160 @@
+- You’ll now be prompted to enter some information about the project you’re about to create: the project and stack name, the configuration details specific to that template, the repo to use when committing your new project code, and optionally the Environment you want to use. Environments enable you to use a bundle of pre-configured secrets and configuration values without needing to re-specify all of them during project creation.
+- After you’ve configured your project settings, you will be able to configure the behavior of Deployments, including when to trigger new Deployments and environment variables to include with your updates.
+
+And just like that you have created a new project from a template that is deployed, without the use of the Pulumi CLI.
+
+### Organization Templates
+
+Pulumi now allows customers on the Pulumi Cloud Enterprise and Business Critical editions to define Organization Templates to help get projects off the ground faster. A Pulumi organization admin can specify an Organization Template Source (or Sources) which will then be populated in the New Project Wizard experience described above.
+
+Navigate to the “Integrations” tab to define your Organization Templates Source. Enter sources as `github.com/<owner>/<repo>/<optional subdirectory>`. A source can be a directory containing either a Pulumi.yaml, or other subdirectories with their own Pulumi.yaml files. A single repository can provide multiple templates from various subdirectories. For example, these are both valid sources: `github.com/pulumi/templates` (all public Pulumi templates) and `github.com/pulumi/templates/aws-typescript` (a specific public template).
+
+The Pulumi.yaml file can optionally contain a template section, which typically includes a config section for specifying required config values for the project. Each config value can have a description and a default value which can be overridden. Config values can be marked as secret, which ensures values in templated projects will be stored with secure encryption.
+
+```yaml
+name: my-aws-project
+runtime: nodejs
+description: My AWS project description
+template:
+  config:
+    aws:region:
+      description: The AWS region to deploy into
+      default: us-west-2
+    myAccessToken:
+      description: My access token
+      secret: true
+```
+
+The above snippet includes an aws:region configuration value with a default value of us-west-2, as well as a myAccessToken secret without a default value.
+
+### Pulumi & Backstage
+
+We’ve seen many developer portal technologies rapidly growing in popularity over the last few years. In particular, we’ve seen Pulumi users adopting [Backstage](https://backstage.io) and as a result we built the Pulumi Backstage Plugin to address the needs of organizations using Backstage and Pulumi together. There are now two ways to use Pulumi from within Backstage:
+Pulumi Scaffolder Backend Module
```

Comment:
Is this like option a, option b? in the preview link both these run into each other. should it be A and B or A or B?